deploy/minikube/schema_check.go

```diff
@@ -45,9 +45,9 @@ func validateSchema(schemaPathString string, docPathString string) {
 	}
 
 	if result.Valid() {
-		fmt.Printf("The document %s is valid\n", docPathString)
+		fmt.Printf("The document %s is valid \n", docPathString)
```

Comment:
Ya, the text is suppose to be printed on stdout and adding space will not matter here.